<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SVPreferencesPageTools</class>
 <widget class="QWidget" name="SVPreferencesPageTools">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>548</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>External Post-Processing</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="1,0">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>PostProc 2 Executable:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QtExt::BrowseFilenameWidget" name="filepathPostProc" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButtonAutoDetectPP2">
        <property name="text">
         <string>Auto-detect</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Text editor</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Executable:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
        <item>
         <widget class="QtExt::BrowseFilenameWidget" name="filepathTextEditor" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonAutoDetectPP2_2">
          <property name="text">
           <string>Auto-detect</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxUseTerminal">
        <property name="text">
         <string>Open in terminal</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelUseTerminal">
        <property name="text">
         <string>Command line for terminal:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditTerminal"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox7Zip">
     <property name="title">
      <string>7-Zip</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Executable:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
        <item>
         <widget class="QtExt::BrowseFilenameWidget" name="filePath7Zip" native="true"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonAutoDetect7zip">
          <property name="text">
           <string>Auto-detect</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxCCMEditor">
     <property name="title">
      <string>Climate data editor</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>CCM Editor Executable:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
        <item>
         <widget class="QtExt::BrowseFilenameWidget" name="filePathCCMEditor" native="true"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonAutoDetectCCM">
          <property name="text">
           <string>Auto-detect</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtExt::BrowseFilenameWidget</class>
   <extends>QWidget</extends>
   <header>QtExt_BrowseFilenameWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>checkBoxUseTerminal</tabstop>
  <tabstop>lineEditTerminal</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
